commit e5ce3c06f00960b860921f205f53623fab01d8c1
Author: Davide Cavalca <davide125@tiscali.it>
Date:   Thu Jul 28 20:42:45 2016 -0700

    yummy:patchX:build-sys--conditionally-disable-LTO-if-requested---3823-.patch

build-sys: conditionally disable LTO if requested (#3823)

    
    This adds a --disable-lto option to ./configure, but does not change the
    default behavior.

diff --git a/configure.ac b/configure.ac
index 4c8d07e..5574970 100644
--- a/configure.ac
+++ b/configure.ac
@@ -211,9 +211,12 @@ AS_CASE([$CC], [*clang*],
                -Wno-gnu-variable-sized-type-not-at-end \
         ])])
 
+AC_ARG_ENABLE([lto], [AS_HELP_STRING([--disable-lto], [disable -flto])],
+                     [], [enable_lto=yes])
 AS_CASE([$CFLAGS], [*-O[[12345sz\ ]]*],
-        [CC_CHECK_FLAGS_APPEND([with_cflags], [CFLAGS], [\
-               -flto])],
+        [AS_IF([test "x$enable_lto" = "xyes"],
+               [CC_CHECK_FLAGS_APPEND([with_cflags], [CFLAGS], [-flto])],
+               [AC_MSG_RESULT([disabling -flto as requested])])],
         [AC_MSG_RESULT([skipping -flto, optimization not enabled])])
 AC_SUBST([OUR_CFLAGS], "$with_cflags $sanitizer_cflags")
 
